commitizen/cz/utils.py

```diff
@@ -6,13 +6,34 @@
 from commitizen.cz import exceptions
 
 
-def required_validator(answer, msg=None):
+def required_validator(answer: str, msg=None) -> str:
```

Comment:
We probably would like to add test cases to validate these validators

Reply:
I know we named it as `required_validator` but would be great if we change them to something like `require_validator`